### PR TITLE
Revert the null value test for Vagrant driver

### PR DESCRIPTION
**Pull request by micheelengronne**
To review with caution. Related to https://github.com/ansible/molecule/issues/2022

#### PR Type

- Bugfix Pull Request

This particular unit test seems broken. I reverted the current behavior for the third time after https://github.com/ansible/molecule/commit/cd2ea98325bd1a418371eb8a24b0e2a1b5952f05#diff-d3601755c38e1ab64638083914501041 and https://github.com/ansible/molecule/commit/bd69c6d9af6bc9683fd3eda74f30d9060de878e7#diff-d3601755c38e1ab64638083914501041

This should be carefully reviewed as it is a personal opinioned patch following https://github.com/ansible/molecule/issues/2022#issuecomment-488613124

### CODE REVIEW

**Comment by decentral1se** [comment]
I think there are two options:

  1) Accept this PR and then bound Cerberus <= 1.3
  2) Bound Cerberus to <=1.2 and then the test should pass

I'm easy with 1 if the minor version bounding is the right approach?

**Comment by micheelengronne** [comment]
As I discovered the molecule code literraly today, I don't know if there is any implications to my MR, hence the caution. So I can't answer to that.

**Comment by decentral1se** [comment]
Sure. Given https://github.com/ansible/molecule/pull/2024#issuecomment-488686654, I think we're good to bound Cerberus to 1.3. It'd be great if you could do that in this PR (or the other). Sorry for all this back and forth now but I think we have the solutions now :)

File: setup.cfg [hunk 1]
```diff
@@ -71,7 +71,7 @@ install_requires =
     ansible-lint >= 4.0.2, < 5
 
     anyconfig == 0.9.7
-    cerberus >= 1.2, < 3
+    cerberus <= 1.3
```
Comment (decentral1se) [inline setup.cfg]: Maybe link to https://github.com/ansible/molecule/issues/2022#issuecomment-488682442 in a comment?

**Comment by decentral1se** [comment]
Since this will fix the broken CI, I'll be merging it when it goes green. 

Please shout before if you have concerns.

**Pull request merged**